packages/fxa-auth-server/config/index.ts

```diff
@@ -387,6 +389,40 @@ const conf = convict({
       default: '',
       env: 'SES_CONFIGURATION_SET',
     },
+    bounces: {
```

Comment:
resurrecting the pre-email-service config from https://github.com/mozilla/fxa-auth-server/pull/1934